.github/workflows/lint.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: Static Analysis
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+
+#############################
+# Start the job on all push #
+#############################
+on: [push]
+
+jobs:
+  static-analysis:
+      # Name the Job
+      name: Lint Code Base
+      # Set the agent to run on
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout Code
+          uses: actions/checkout@v2
+          with:
+            # Full git history is needed to get a proper list of changed files within `super-linter`
+            fetch-depth: 0
+
+        - name: Lint Code Base
+          uses: github/super-linter@v4
+          env:
+            VALIDATE_ALL_CODEBASE: false
+            DEFAULT_BRANCH: main
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            FILTER_REGEX_EXCLUDE: 
```

Comment:
Seems like the default is none. Should we comment this line out?

Reply:
I think it has the same effect: empty or commented out. I didn't notice any issues when it ran.